tests/components/trafikverket_ferry/test_init.py

```diff
@@ -0,0 +1,60 @@
+"""Test for Sensibo component Init."""
+from __future__ import annotations
+
+from unittest.mock import patch
+
+from pytrafikverket.trafikverket_ferry import FerryStop
+
+from homeassistant import config_entries
+from homeassistant.components.trafikverket_ferry.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+
+from . import ENTRY_CONFIG
+
+from tests.common import MockConfigEntry
+
+
+async def test_setup_entry(hass: HomeAssistant, get_ferries: list[FerryStop]) -> None:
+    """Test setup entry."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        source=SOURCE_USER,
+        data=ENTRY_CONFIG,
+        entry_id="1",
+        unique_id="123",
+    )
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.trafikverket_ferry.coordinator.TrafikverketFerry.async_get_next_ferry_stops",
+        return_value=get_ferries,
+    ):
```

Comment:
Maybe check the mock is called, although it may be unnecessary since it's validated by the sensor tests